VideoLocker/src/main/java/org/edx/mobile/module/notification/ParseSyncTask.java

```diff
@@ -0,0 +1,36 @@
+package org.edx.mobile.module.notification;
+
+import android.content.Context;
+
+import com.parse.ParseInstallation;
+
+import org.edx.mobile.model.api.AuthResponse;
+import org.edx.mobile.task.Task;
+
+import java.util.List;
+
+/**
+ * Sync with Parse server
+ */
+public abstract class ParseSyncTask extends Task<Void> {
```

Comment:
Instead of a ``Void`` Result argument, should this have a ``List<String>`` Result?